src/game/select_cards.rs

```diff
@@ -0,0 +1,143 @@
+use crate::game::cards::*;
+use crate::game::scoring::*;
+use itertools::Itertools;
+
+/**
+ * go through each of the 16 combinations looking for the hand
+ * that will perform best based on the value of the hand plus
+ * or minus the value of the crib
+ */
+pub fn select_crib_cards(hand: &[Card], is_my_crib: bool) -> Vec<Card>{
+
+    // get all possible hands
+    let local_hand = hand.to_vec();
```

Comment:
is local_hand use anywhere other than the following line?

Reply:
Good call, removed the local var and moved to_vec inline.